spec/restaurant_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'restaurant'
+
+describe Restaurant do
+  subject(:restaurant) { described_class.new }
```

Comment:
Whats the benefit in using described_class rather than Restaurant?
Question for you as I dont know the answer!